spark/v3.1/spark/src/main/scala/com/netease/arctic/spark/ArcticSparkExtensions.scala

```diff
@@ -47,9 +47,9 @@ class ArcticSparkExtensions extends (SparkSessionExtensions => Unit) {
     extensions.injectOptimizerRule { spark => RewriteUpdate(spark) }
     extensions.injectOptimizerRule { spark => RewriteMergeInto(spark) }
     // arctic optimizer rules
-    extensions.injectOptimizerRule { spark => RewriteAppendArcticTable(spark) }
-    extensions.injectOptimizerRule { spark => RewriteDeleteFromArcticTable(spark) }
-    extensions.injectOptimizerRule { spark => RewriteUpdateArcticTable(spark) }
+//    extensions.injectOptimizerRule { spark => RewriteAppendArcticTable(spark) }
```

Comment:
Why remove these rules?

Reply:
next version's function, now it does not work